Cargo.toml

```diff
@@ -16,41 +16,54 @@ path = "src/main.rs"
 
 [features]
 STM32H745ZITx = []
+0483 = []                                             # CONFIGURE: Rename with target PID for your probe
+374e = []                                             # CONFIGURE: Rename with target VID for your probe
 nRF52833_xxAA = ["nrf52833-hal"]
+0d28 = []                                             # CONFIGURE: Rename with target PID for your probe
+0204 = []                                             # CONFIGURE: Rename with target VID for your probe
 RP2040 = ["embedded-time", "rp-pico", "embedded-hal"]
+cafe = []                                             # CONFIGURE: Rename with target PID for your probe
+4005 = []                                             # CONFIGURE: Rename with target VID for your probe
 esp32c3 = ["esp32c3-hal"]
+303a = []                                             # CONFIGURE: Rename with target PID for your probe
+1001 = []                                             # CONFIGURE: Rename with target VID for your probe
 
 
 [target.thumbv6m-none-eabi.dependencies]
-cortex-m = { version = "0.7" }
-cortex-m-rt = { version = "0.7" }
+cortex-m = { version = "0.7.6" }
+cortex-m-rt = { version = "0.7.1" }
 rtt-target = { version = "0.3.1", features = ["cortex-m"] }
-panic-probe = { version = "0.2", features = ["print-rtt"] }
+panic-probe = { version = "0.3", features = ["print-rtt"] }
 
 [target.thumbv7em-none-eabihf.dependencies]
-cortex-m = { version = "0.7" }
-cortex-m-rt = { version = "0.7" }
+cortex-m = { version = "0.7.6" }
```

Comment:
we should never include patch deps. Those are added by cargo automatically. We only add those if we want to fix the version to a specific patch.

Reply:
OK. I've left it in as a comment, so that it is easy for me to access when I need to test.